packages/bpk-component-datatable/src/BpkDataTable.js

```diff
@@ -139,6 +139,9 @@ class BpkDataTable extends Component {
     if (this.state.rowSelected === index) {
       classNames.push(getClassName('bpk-data-table__row--selected'));
     }
+    if (this.props.onRowClick !== undefined) {
```

Comment:
In case they provide `null` for some reason

```suggestion
    if (this.props.onRowClick) {
```

Reply:
I did it this way to maintain parity with the logic in `onRowClicked`:

```js
    if (this.props.onRowClick !== undefined) {
      this.props.onRowClick(this.state.sortedList[index]);
    }
```

We could change it there too but I worry it'll have unforeseen consequences.

Reply:
If that's already a limitation there then this is fine.
If they pass in null (or any other non-function except `undefined`) then this will blow up though

Reply:
I wonder why we did it this way in the first place 🤔